execution/executor/src/lib.rs

```diff
@@ -310,13 +305,21 @@ where
         // Only bump the counter when the commit succeeds.
         OP_COUNTERS.inc_by("num_accounts", list_num_account_created.into_iter().sum());
 
+        // Calculate committed transactions and reconfig events now that commit has succeeded
+        let mut committed_txns = vec![];
+        let mut reconfig_events = vec![];
+        for txn in txns_to_commit.iter() {
+            committed_txns.push(txn.transaction().clone());
```

Comment:
Is this clone unnecessary? (can we return `txns_to_commit`)?

Reply:
unfortunately `Transaction` doesn't implement `Copy` trait - that is why clone is necessary